cnc/server/telemetry_engine.py

```diff
@@ -44,8 +45,15 @@ def updateDroneStatus(self, extras):
                     {"latitude": extras.location.latitude, "longitude": extras.location.longitude, "altitude": extras.location.altitude,
                      "rssi": extras.status.rssi, "battery": extras.status.battery, "mag": extras.status.mag, "bearing": int(extras.status.bearing)},
                 )
+
+        drone_key = f"drone:{extras.drone_id}"
         logger.debug(f"Updated status of {extras.drone_id} in redis under stream telemetry at key {key}")
 
+        self.r.hset(drone_key, "last_seen", f"{time.time()}")
+        self.r.hset(drone_key, "battery", f"{extras.status.battery}")
+        self.r.expire(drone_key, self.DRONE_TTL_SECS)
+        logger.debug(f"Updating drone status: last_seen: {time.time()}")
```

Comment:
The log statement should probably include which drone id you are updating the last seen for.